dtm0/service.c

```diff
@@ -306,6 +303,21 @@ M0_INTERNAL bool m0_dtm0_is_a_persistent_dtm(struct m0_reqh_service *service)
 		to_dtm(service)->dos_origin == DTM0_ON_PERSISTENT;
 }
 
+M0_INTERNAL int m0_dtm0_service_find(struct m0_reqh *reqh,
+				     struct m0_dtm0_service **out)
+{
+	struct m0_reqh_service *rh_srv;
+
+	rh_srv = m0_reqh_service_find(&dtm0_service_type, reqh);
+	if (rh_srv == NULL)
```

Comment:
m0_reqh_service_find() signature in service.h returns an int. Should this be: reqh == NULL ? or rather in keeping with our coding style: if(!reqh)